### PR TITLE
Test/klbp 293

### PR DESCRIPTION
**Pull request by mariuspop86**

### CODE REVIEW

File: src/components/Journeys/Common/DisclaimerSection.tsx [hunk 1]
```diff
@@ -4,20 +4,15 @@ import DynamicFormattedMessage from '../../common/DynamicFormattedMessage'
 
 const DisclaimerSection: FC = () => {
   return (
-    <InformationBox className="bg-warning-light mb-20" iconPosition="start">
-      <div className="mb-2 leading-6 text-md">
+    <InformationBox className="bg-warning-light">
```
Comment (iuliudascalu) [inline src/components/Journeys/Common/DisclaimerSection.tsx:7]: This conflict is not solved correctly, please revert the changes.

File: src/hooks/useStepFormHook.ts [hunk 2]
```diff
@@ -1,69 +1,96 @@
...
 
     dispatch(folderActions.folderLoading())
+    dispatch(breadcrumbActions.breadcrumbLoading())
     const { action, ...inputData } = data
 
     const formData: IFormData = {
       data: inputData,
       step,
-      next_step: getNextStep(),
+      next_step: step + 1,
```
Comment (ioan-iacob-netea) [inline src/hooks/useStepFormHook.ts:75]: Please revert this change.

File: src/hooks/useStepFormHook.ts [hunk 3]
```diff
@@ -1,69 +1,96 @@
...
     }
 
     try {
       await folder.saveFolder(formData)
+      // TODO: Move patchBreadcrumbs to KYC-Core
```
Comment (ioan-iacob-netea) [inline src/hooks/useStepFormHook.ts:80]: TODO no longer required as the patch was moved to KYC-Core.

File: src/hooks/useStepFormHook.ts [hunk 4]
```diff
@@ -78,7 +105,7 @@ const useStepFormHook = () => {
     isDisabled: status === STATUS.pending,
   }
 
-  return { form, onSubmit, NavButtonsProps }
+  return { form, onSubmit, NavButtonsProps, patchBreadcrumbs }
```
Comment (ioan-iacob-netea) [inline src/hooks/useStepFormHook.ts:108]: patchBreadcrumb from useStepFormHook was moved to useBreadcrumbsHook and used from there.

File: src/hooks/useStepFormHook.ts [hunk 5]
```diff
@@ -1,69 +1,96 @@
...
+    ) {
+      history.push(ROUTES.confirmation)
+    }
+
+    if (actualStep !== step) {
+      history.push(ROUTES[`step${step}`])
+    }
+  }, [step, journeyStatus])
+
+  const patchBreadcrumbs = async (isPrevious: boolean = false) => {
```
Comment (ioan-iacob-netea) [inline src/hooks/useStepFormHook.ts:47]: This method moved to useBreadcrumbsHook and is no longer required here, just the call from a few lines bellow.

File: src/enums/common.ts [hunk 6]
```diff
@@ -66,22 +80,3 @@ export const STATUS_VERIFICATION_ALLOW_UPLOAD_RETRY = [
...
-  JDD: '50',
-  JUSTIFICATIF_REVENU: '131',
-  FORMULAIRE_AUTO_CERTIFICATION: '70',
-}
-
-export const ALLOW_FILE_TYPES = ['image/jpeg', 'image/png', 'application/pdf']
-export const ALLOW_SIZE = 20971520 // 20MB
-
-export const NEXT_PAGE = 'next'
-export const PREVIOUS_PAGE = 'previous'
```
Comment (ioan-iacob-netea) [inline src/enums/common.ts:87]: The above two constants are required on the breadcrumbs flow and should not be removed.

File: src/hooks/useStep2Hook.ts [hunk 7]
```diff
@@ -153,12 +151,12 @@ const useStep2Hook = () => {
...
 
-  const onClickPrevious = async (): Promise<void> => {
-    dispatch(loadingActions.loading())
-    await patchBreadcrumbs(PREVIOUS_PAGE)
-    await getBreadcrumbs()
-    history.push(ROUTES.step0) // TODO this should be dynamic, for EI this should be enough
-    dispatch(loadingActions.loaded())
+  const onClickPrevious = async (isPrevious?: boolean): Promise<void> => {
+    dispatch(breadcrumbActions.breadcrumbLoading())
+    await patchBreadcrumbs(isPrevious)
```
Comment (ioan-iacob-netea) [inline src/hooks/useStep2Hook.ts:156]: Please use patchBreadcrumbs from useBreadcrumbsHook.

**Review by ioan-iacob-netea — CHANGES_REQUESTED** [review]

File: src/components/Journeys/IndividualCompany/Step1/Step1Content.tsx [hunk 8]
```diff
@@ -10,7 +10,7 @@ function Step1Content(): ReactElement {
   return (
     <>
       <Step1InfoSection existingDocuments={existingDocuments} />
-      <Step1Form {...step1HookData} />
```
Comment (iuliudascalu) [inline src/components/Journeys/IndividualCompany/Step1/Step1Content.tsx:13]: This conflict is not solved correctly, please revert the changes.

File: src/components/Journeys/IndividualCompany/Step1/Step1InfoSection.tsx [hunk 9]
```diff
@@ -12,60 +12,62 @@ function Step1InfoSection({
 }): ReactElement {
   const nom = useSelector((state: IStore) => state.folderReducer.folder.nom)
   return (
-    <div className="xl:w-9/12 xl:pr-28">
-      {existingDocuments && <DisclaimerSection />}
-      <h3 className="subtitle mb-4">{nom}</h3>
-      <div className="text-default mb-6">
-        <DynamicFormattedMessage id="step1.text1" tag="p" />
-        <DynamicFormattedMessage id="step1.text2" tag="p" />
+    <>
```
Comment (iuliudascalu) [inline src/components/Journeys/IndividualCompany/Step1/Step1InfoSection.tsx:15]: This conflict is not solved correctly, the fragment is not longer needed.

File: src/components/Steps/Step2Content.tsx [hunk 10]
```diff
@@ -1,16 +1,16 @@
...
 import { IStore } from '../../interfaces/Store'
-import { CLIENT_TYPOLOGY } from '../../enums/journeys'
+import { JOURNEYS } from '../../enums/journeys'
+import { ROUTES } from '../../enums/routes'
 
 const Step2Content = () => {
   const { typologie_client } = useSelector(
     (state: IStore) => state.folderReducer.folder
   )
-  const { INDIVIDUAL_COMPANY } = CLIENT_TYPOLOGY
```
Comment (iuliudascalu) [inline src/components/Steps/Step2Content.tsx:13]: This conflict is not solved correctly, please revert the changes.

File: src/components/Steps/Step1Content.tsx [hunk 11]
```diff
@@ -1,16 +1,16 @@
...
+import { JOURNEYS } from '../../enums/journeys'
 import IndividualCompanyStep1Content from '../Journeys/IndividualCompany/Step1/Step1Content'
-import { ROUTES } from '../../enums/routes'
 
 const Step1Content = () => {
   const { typologie_client } = useSelector(
     (state: IStore) => state.folderReducer.folder
   )
-  const { INDIVIDUAL_COMPANY } = CLIENT_TYPOLOGY
+  const { INDIVIDUAL_COMPANY } = JOURNEYS
```
Comment (iuliudascalu) [inline src/components/Steps/Step1Content.tsx:13]: This conflict is not solved correctly, please revert the changes.
